<ui version="4.0" >
 <class>AutoSMPModelerWidget</class>
 <widget class="QWidget" name="AutoSMPModelerWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>10</y>
     <width>301</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Number of discretizationsteps</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>51</width>
        <height>27</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2" >
     <widget class="QSpinBox" name="numdstepsEdit" >
      <property name="minimum" >
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Number of states</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <widget class="QSpinBox" name="numstatesEdit" >
      <property name="minimum" >
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Confidence niveau</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" >
     <widget class="QSpinBox" name="confniveauEdit" >
      <property name="minimum" >
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>120</y>
     <width>301</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>261</width>
        <height>41</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" >
     <widget class="QPushButton" name="computeButton" >
      <property name="text" >
       <string>Compute</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
